src/Umbraco.Web.UI.Client/src/less/listview.less

```diff
@@ -243,7 +243,8 @@
   background-color: @grayLighter
 }
 
-.table-striped tbody i:hover {
+/* don't hide all icons, e.g. for a sortable handle */
+.table-striped tbody i:not(.handle):hover {
   display: none !important
 }

```

Comment:
This is important as it affected "handle" icons in the table for related links, where they disappeared on hover. It could also be overwritten in the less file for tables I guess by setting it to display:inline https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements .. but it need to use ```!important``` too then..